pyproject.toml

```diff
@@ -25,17 +23,16 @@ pathvalidate = {version = "^3.2.0", source = "PyPI"}
 pre-commit = {version = "^3.2.0", source = "PyPI"}
 py360convert-modified = {version = "^1.0.1", source = "PyPI"}
 pydantic = {version = "^1.8.2", source = "PyPI"}
-torch = { version = "^2.2.1", source="torch"}
-torchaudio = { version = "^2.2.1", source="torch"}
-torchvision = { version = "^0.17.1", source="torch"}
-ultralytics = "^8.1.35"
-
-[[tool.poetry.source]]
-name = "torch"
-url = "https://download.pytorch.org/whl/cu118"
-priority = "explicit"
+torch = {version = "2.2.2", source = "PyPI"}
```

Comment:
These are probably causing the GPU issues. Perhaps we can work with two different .toml files, one for Luna and one for Azure? Or even two repos...

Reply:
For now solved by using a different requirements.txt for Luna